fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/dao/api/ResourceDAO.java

```diff
@@ -84,6 +84,16 @@ int historyCount(String resourceType, String logicalId, Timestamp fromDateTime)
      * @throws FHIRPersistenceDBConnectException
      */
     List<Resource> search(SqlQueryData queryData) throws FHIRPersistenceDataAccessException, FHIRPersistenceDBConnectException;
+
+    /**
+     * Executes the search contained in the passed SqlQueryData, using it's encapsulated search string and bind variables.
+     * @param queryData - Contains a search string and (optionally) bind variables.
+     * @return List<String> A list of strings satisfying the passed search.
+     * @throws FHIRPersistenceDataAccessException
+     * @throws FHIRPersistenceDBConnectException
+     */
+    List<String> searchSTR_VALUES(SqlQueryData queryData) throws FHIRPersistenceDataAccessException, FHIRPersistenceDBConnectException;
```

Comment:
are you sure its needed on the interface?
if so, how does it differ from `search(SqlQueryData)`?  the javadoc needs to make this clear.

Reply:
If it is really need, then I suggest renaming it to searchStringValues or something that is more consistent with the naming scheme we have throughout the codebase

Reply:
this is really needed, because only the interface was passed into the function which I call the function from.

and OK, let me change the name of it.

Reply:
done!